### PR TITLE
Use query syntax instead of foreach

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/2592

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: examples/facebook/posts/posts.bal [hunk 1]
```diff
@@ -52,9 +52,10 @@ service /facebook on fbListener {
     resource function get posts() returns string[]|error {
```
Comment (niveathika) [inline examples/facebook/posts/posts.bal:52]: Lets mark these resources as isolated. Better check in other places as well.

Comment (kaneeldias) [reply examples/facebook/posts/posts.bal]: Fixed in 3fa5fad and 92b35d4

File: examples/snowflake/README.md [hunk 2]
```diff
@@ -83,9 +83,10 @@ _ = check dbClient->execute(`
 `);
 
 stream<record{}, sql:Error?> streamData = dbClient->query("SELECT * FROM Employees");
-_ = check streamData.forEach(function(record {} data) {
+from record{} data in streamData
```
Comment (niveathika) [inline examples/snowflake/README.md:86]:
```suggestion
check from record{} data in streamData
```

Comment (kaneeldias) [reply examples/snowflake/README.md]: Fixed in 816fd90

**Review by niveathika — APPROVED** [review]

**Pull request merged**